mcodingbot/plugins/stats.py

```diff
@@ -150,3 +153,21 @@ def display_stats(stat: int | float) -> str:
     # gonna be such a problem if it's gonna be shown as e.g. "44.3K"
 
     return f"2**{exp_stat} ({pretty_stat}{unit})"
+
+
+@plugin.include
+@crescent.command(
+    name="stats", description="Exact values for mCoding statistics"
+)
+async def stats(ctx: Context) -> None:
+    stats = await get_stats(ctx.bot)
```

Comment:
Stats should not be fetched every time the command is used. They should be cached when the function called `loop` is run and fetched later. Is is too prevent API ratelimits.

Reply:
I believe you can use the _last_known_stats variable (it's a global in this file)

Reply:
Also I think the command should be at the top of the file rather than the bottom. cc @Lunarmagpie @trag1c

Reply:
Agreed

Reply:
I don't mind to be fair